core/css.js

```diff
@@ -154,6 +154,7 @@ Blockly.Css.CONTENT = [
     'height: 100%;',
     'position: relative;',
     'overflow: hidden;', /* So blocks in drag surface disappear at edges */
+    'touch-action: none',
```

Comment:
It just suppresses the warnings about preventDefault in a passive handler.  It could go into develop or here--it doesn't seem to change the behaviour other than suppressing warnings, presumably because we're manually stopping the default actions anyway.